src/type/AccessBaseOptions.ts

```diff
@@ -19,13 +19,17 @@
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
+/**
+ * @module interfaces
+ */
```

Comment:
An open question @NSeydoux & @matthieubosquet is: Do we want the interfaces to be located in `src/type/` or would we be better with them in `src/interfaces/` ?

Reply:
I don't have a strong opinion, in fact I'm not even sure having them in a separate module is what we want. Do you have a preference between the two options ?